Macros/Macros.js

```diff
@@ -223,64 +393,6 @@ class FurnaceMacros {
         div.find(".furnace-macro-expand").on('click', (ev) => div.toggleClass("fullscreen"));
         refreshHighlight();
     }
-
-    static async _execute(message) {
-        let macro = game.macros.getName(message.macroName);
-        if (!macro || !getProperty(macro.data, "flags.furnace.gmexecute" || !game.users.get(macro.data.author).isGM)) {
-            ui.notification.error(`User ${message.userId} tried to invailidly GM execute macro ${message.macroName}`);
-            return "Invalid Request";
-        }
-
-        try {
-            return await macro.renderContent("RemoteContext", message, ...message.args);
-        } catch (err) {
-            ui.notifications.error(`There was an error in your macro syntax. See the console (F12) for details`);
-            console.error(err);
-            return "Macro syntax error";
-        }
-    }
-
-    // request remote execution of a macro
-    static async requestRemoteExecute(macroName, ...args) {
-        let targetGM = game.users.entities.find(u => u.isGM && u.active);
-        let macroMessage = {
-            action: "GMExecuteMacro", 
-            requestId: "remoteResult" + game.user.id + Date.now(), 
-            userId: game.user.id, 
-            targetGM: targetGM._id, 
-            macroName, 
-            sceneId: canvas.scene.id, // pass this so macros can know which scene the player is on.
-            tokenId: ChatMessage.getSpeaker().token, 
-            event: event || {event: {shiftKey: true}}, // this is to keep minor-qol happy
```

Comment:
probably should just be event,
event is set if the macro is run via the hotbar. the {event: ....} should be removed.